resources/js/hiroapp/components/recruiter/CreatePosition.jsx

```diff
@@ -0,0 +1,123 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const CreatePosition = () => {
+	const navigate = useNavigate;
```

Comment:
calling the hook was missing, as I am familiar with typos it took time for me to see as well :)
const navigate = useNavigate();